Docs/source/install/hpc/pitzer.rst

```diff
@@ -0,0 +1,218 @@
+
+.. _building-pitzer-update:
+
+Update WarpX & Dependencies
+---------------------------
+
+If you already installed WarpX in the past and want to update it, start by getting the latest source code:
+
+.. code-block:: bash
+
+   cd $HOME/src/warpx
+
+   # read the output of this command - does it look ok?
+   git status
+
+   # get the latest WarpX source code
+   git fetch
+   git pull
+
+   # read the output of these commands - do they look ok?
+   git status
+   git log # press q to exit
+
+And, if needed,
+
+- :ref:`update the pitzer_warpx.profile file <building-pitzer-preparation>`,
+- log out and into the system, activate the now updated environment profile as usual,
+- :ref:`execute the dependency install scripts <building-pitzer-preparation>`.
+
+As a last step, clean the build directory ``rm -rf $HOME/src/warpx/build_*`` and rebuild WarpX.
+
+.. _running-pitzer:
+
+Running
+-------
+
+.. tab-set::
+
+   .. tab-item:: GPU Nodes
+
+      Pitzer's GPU partition includes:
+
+      - 32 nodes, each equipped with two V100 (16GB) GPUs.
+      - 42 nodes, each with two V100 (32GB) GPUs.
+      - 4 large memory nodes, each with quad V100 (32GB) GPUs.
+
+      To run a WarpX simulation on the GPU nodes, use the batch script provided below. Adjust the ``-N`` parameter in the script to match the number of nodes you intend to use. Each node in this partition supports running one MPI rank per GPU.
+
+      .. literalinclude:: ../../../../Tools/machines/pitzer-osc/pitzer_v100.sbatch
+         :language: bash
+         :caption: Copy this file from ``$HOME/src/warpx/Tools/machines/pitzer-osc/pitzer_v100.sbatch``.
+
+      After preparing your script, submit your job with the following command:
+
+      .. code-block:: bash
+
+         sbatch pitzer_v100.sbatch
+
+   .. tab-item:: CPU Nodes
+
+      For CPU-based computations, Pitzer offers:
+
+      - 224 nodes, each with dual Intel Xeon Gold 6148 CPUs and 192 GB RAM.
+      - 340 nodes, each with dual Intel Xeon Platinum 8268 CPUs and 192 GB RAM.
+      - 16 large memory nodes.
+
+      To submit a job to the CPU partition, use the provided batch script. Ensure you have copied the script to your working directory.
+
+      .. literalinclude:: ../../../../Tools/machines/pitzer-osc/pitzer.sbatch
+         :language: bash
+         :caption: Copy this file from ``$HOME/src/warpx/Tools/machines/pitzer-osc/pitzer.sbatch``.
+
+      Submit your job with:
+
+      .. code-block:: bash
+
+         sbatch pitzer.sbatch
+
+.. _post-processing-osc:
+
+Post-Processing
+---------------
+
+.. note::
+   This section is a stub and contributions are welcome.
```

Comment:
Missing newline.
```suggestion

   This section is a stub and contributions are welcome.
```

Do you want to link the OSC Jupyter documentation here again or add more instructions (see, e.g., Perlmutter docs).